Assets/Scripts/Controllers/ScoreController.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ScoreController : MonoBehaviour
+{
+    public static ScoreController scInstance = null;
+
+    private int currentScore;
+
+    private int highScore;
+
+    private string HIGH_SCORE_KEY = "HighScore";
+
+    void Awake()
+    {
+        if (scInstance == null)
+        {
+            scInstance = this;
+        }
+        else if (scInstance != this)
+        {
+            Destroy(gameObject);
+        }
+
+        currentScore = 0;
+        highScore = LoadScore();
+    }
+
+    private int LoadScore()
```

Comment:
Should `LoadScore` just load the value into `highScore`? I don't see the need to return it

Reply:
oooo yeah agree - will fix